tox/config.py

```diff
@@ -219,8 +219,11 @@ def parseconfig(args=None, plugins=()):
 
     # parse ini file
     basename = config.option.configfile
-    if os.path.isabs(basename):
+    if os.path.isfile(basename):
```

Comment:
No problem here - isfile also tru if it is a symlink: https://docs.python.org/2/library/os.path.html#os.path.isfile